Samples/CameraSettingsExternalSettingsApp/readme.md

```diff
@@ -1,9 +1,12 @@
 # ExternalCameraSettingsApp Sample
 
-This sample app will demonstrate how to build an external camera settings app.
+This sample app will demonstrate how to build an external camera settings app associated to a camera device. This app allows to adjust control in real time as well as set a control value as the default one to be applied every time the associated camera is started. The association between app and camera device is typically done by the camera driver at installation time (i.e. by a camera manufacturer wanting to offer an app tie-in to their physical device or a virtual camera implemented offering an app such as [the ***VirtualCameraManager_App*** in our other sample](..\VirtualCamera\README.md)). This settings app would then be launchable directly from the associated camera device settings page *(Windows Settings &rarr; Bluetooth & devices &rarr; Cameras &rarr; \<the associated camera\> in the "Related settings" section)*
 
 This app will use camera symbolic link(symlink) name that's provided via [Application.OnLaunched](https://docs.microsoft.com/uwp/api/windows.ui.xaml.application.onlaunched?view=winrt-22000) arguments and typically passed further on to the [Page.OnNavigatedTo](https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.page.onnavigatedto?view=winrt-22000). The camera symlink is only passed to the app launch when the app is registered to be an additional Settings app for the specific camera.
 
+## Requirements
+This sample is built using Visual Studio 2019 and requires [Windows SDK version 22621](https://developer.microsoft.com/en-us/windows/downloads/windows-sdk/).

```

Comment:
> 2019 [](http://example.com/codeflow?start=40&length=5)

2019 or 2022 or both ?
I'm using 2022

Reply:
I think they all target vc142

Reply:
so 2019 is sufficient (by default aligns with vc142), I bult them fine with 2019